composer/core/time.py

```diff
@@ -944,4 +973,29 @@ def ensure_time(maybe_time: Union[Time, str, int], int_unit: Union[TimeUnit, str
     Returns:
         Time: An instance of :class:`.Time`.
     """
-    return Time.from_input(maybe_time, int_unit)
+    time_obj = Time.from_input(maybe_time, int_unit)
+    return time_obj
+
+
+def verify_wct(timestamp: str) -> Optional[str]:
+    """Return a valid datetime formated wct timestamp if input is a valid wct.
+
+    Args:
+        timestamp (str): A string that represents a timestamp in wct.
+
+    Returns:
+        str: a properly formatted datetime if input is valid else None
+    """
+    if 'h' not in timestamp:
+        timestamp = '0h' + timestamp
+    if 'm' not in timestamp:
+        timestamp = timestamp.replace('h', 'h0m')
+    if 's' not in timestamp:
+        timestamp = timestamp + '0s'
+
+    pattern = r'^(\d+h)?(\d+m)?(\d+s)?$'
+    match = re.match(pattern, timestamp)
+    if bool(match):
+        return timestamp
+    else:
+        return None
```

Comment:
Can we raise error here and give some description of you tried to pass X, we changed it to Y, it doesnt fit eht format XXYYZZ plz fix?